src/platforms/gbm-kms/server/kms/real_kms_output_container.cpp

```diff
@@ -55,6 +57,12 @@ void mgg::RealKMSOutputContainer::update_from_hardware_state()
                     drm_fd == candidate->drm_fd();
             });
 
+        if (connector->connection != DRM_MODE_CONNECTED)
+        {
+            mir::log_warning("Connector is not connected, so not adding this output to the list.");
```

Comment:
I think this is "info" at best (nothing is wrong), and probably not useful at all without some way to identify which output.